src/parser/models/parser.rs

```diff
@@ -0,0 +1,96 @@
+use pest::Parser;
```

Comment:
I would move this file into the mod.rs so that way it would be 
`parser::Parser` (like main/default export) instead of
`parser::models::Parser`

Reply:
@liam0215  for next pr

Reply:
@astherath That's a good point, nesting a single struct/method that deep is ugly